### DISCUSSION TITLE
Adding option to simply unlock accounts ?

### DISCUSSION THREAD

**Discussion by Loiseau2nuit** [ltb-project/self-service-password#785] [Ideas]
Hi there, and first, thanks a lot for all this work !

Is there any way (or any variable i might have missed in the config file) to let users simply unlock their Active Directory account (without having to change their password evrytime) ? 

Thanks in advance

**Comment by coudot** [comment]
This does not exist yet, you can open a new feature request on https://github.com/ltb-project/self-service-password/issues

If you want to contribute, you can also propose a pull request

**Reply by Loiseau2nuit** [reply]
Thanks for your answer.
FR Done. Honnestly i'm not sure i'm able to code it myself ^^
